lisa/tools/azsecd.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from lisa.executable import Tool
+
+
+class Azsecd(Tool):
+    @property
+    def command(self) -> str:
+        return "azsecd"
+
+    @property
+    def can_install(self) -> bool:
+        return False
+
+    def install(self) -> bool:
+        package_name = "azure-security"
```

Comment:
does it need enable extra repo for this package?

Reply:
This is mainly used for Mariner, hence the repo is already present for the package